rust/src/tests/chflags.rs

```diff
@@ -75,6 +75,7 @@ fn get_flags(ctx: &TestContext) -> (FileFlag, FileFlag, FileFlag) {
 
 crate::test_case! {
     /// chflags(2) set the flags provided for the file.
+    // chflags/00.t
```

Comment:
I actually think it's still useful to leave these comments in place.

Reply:
I agree, that's why I added those which were missing.